components/movementsensor/replay/replay_test.go

```diff
@@ -584,38 +734,11 @@ func TestReplayMovementSensorConfigValidation(t *testing.T) {
 	}
 }
 
-func TestReplayMovementSensorProperties(t *testing.T) {
```

Comment:
These tests have been embedded in `TestReplayMovementSensorFunctions`, which is why I deleted this test.